src/js/view/view.js

```diff
@@ -98,7 +98,7 @@ function View(_api, _model) {
     }
 
     function fosMobileBehavior() {
-        return _isMobile && !deviceIsLandscape();
+        return _isMobile && !deviceIsLandscape() && !_model.get('fullscreen');
```

Comment:
> Disable the floating player when in portrait fullscreen on mobile

This will prevent switching float state on mobile when in portrait **or** fullscreen.